EnglishFilter/sections/antiadblock.txt

```diff
@@ -434,7 +434,7 @@ tubsexer.com,homegrownfreaks.net,pornfd.com,bobs-tube.com,pornsexer.com#@#.ads-i
 !#endif
 !
 ! ai_front anti-adblock
-elektrikmen.com,world4.eu,doctorrouter.ru,gcertificationcourse.com,sagliktais.com,1ahaber.com,scubidu.eu,politics-dz.com,game-owl.com,freebulksmsonline.com,fantasyfootballgeek.co.uk,4allprograms.me,unityassets4free.com,hblinks.pro,hdhub4u.cc,exbulletin.com,apkmaza.net,janusnotes.com,tekfame.com,hunter-xhunter.com,sysnettechsolutions.com,wristreview.com,freewebcart.com,parzibyte.me,ftuapps.dev,sznlink.xyz,hinjakuhonyaku.com,toppremiumpro.com,mashtips.com,dramahd.me,gaypornmasters.com#%#//scriptlet("abort-on-property-write", "ai_front")
+shortzzy.*,elektrikmen.com,world4.eu,doctorrouter.ru,gcertificationcourse.com,sagliktais.com,1ahaber.com,scubidu.eu,politics-dz.com,game-owl.com,freebulksmsonline.com,fantasyfootballgeek.co.uk,4allprograms.me,unityassets4free.com,hblinks.pro,hdhub4u.cc,exbulletin.com,apkmaza.net,janusnotes.com,tekfame.com,hunter-xhunter.com,sysnettechsolutions.com,wristreview.com,freewebcart.com,parzibyte.me,ftuapps.dev,sznlink.xyz,hinjakuhonyaku.com,toppremiumpro.com,mashtips.com,dramahd.me,gaypornmasters.com#%#//scriptlet("abort-on-property-write", "ai_front")
```

Comment:
Please provide at least one working link on which this rule is necessary other than `shortzzy.in`
If you will not find any, then please change `shortzzy.*` to `shortzzy.in` and please add it also here:
https://github.com/AdguardTeam/AdguardFilters/blob/6a95275668ae23d3a2dddc13f3d94ac670c0de62/EnglishFilter/sections/antiadblock.txt#L439

Reply:
Ok done.